wiki/Music_theory/Rhythm/fr.md

```diff
@@ -0,0 +1,14 @@
+---
+stub: true
+---
+
+# Rythme
+
+Le **rythme** est "le modèle des sons, des silences et des accents dans une musique" [[1]][r]. Lorsqu'ils créent des [beatmaps](/wiki/Beatmap), les mappeurs prêtent attention à l'emplacement des sons dans la musique afin de déterminer où placer les [objets](/wiki/Hit_object) et les [hitsounds](/wiki/Beatmapping/Hitsound), afin de suivre le rythme et d'en accentuer certaines parties. Comme il s'agit d'un jeu de rythme, le rythme est une partie indissociable de toute beatmap dans osu!.
```

Comment:
adding the french source seems fine enough, but you're still citing the english one. i suggest swapping the references at the bottom, so we won't need to change the number here

Reply:
>  adding the french source seems fine enough, but you're still citing the english one. 

I keept english reference because both sources are similar but the english seem to have different chapters and can help for better understanding I think.

>i suggest swapping the references at the bottom, so we won't need to change the number here

you want 1. english one and 2. french ?

Reply:
yes, since the citation is a translation of the english source

Reply:
> yes, since the citation is a translation of the english source

its not exactly a translation even if both are on the same topic it doesn't explain it with the same method but I can swap the references as well

Reply:
that's not what i mean

"le modèle des sons, des silences et des accents dans une musique" is a translation of "the pattern of sound, silence, and emphasis in a song", which is a citation of the english source, meaning it cites the english source, not the french one (it doesn't appear there).

Reply:
I understand now, in my mind I didn't link the citation with the source, I will swap them